create_dmg.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [[ "$SHOULD_BUILD" == "yes" ]]; then
+  if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+    cd VSCode-darwin
+    create-dmg VSCodium-darwin-${LATEST_MS_TAG}.app ../
```

Comment:
Here is the command with the correct syntax.

Reply:
The app file here actually doesn't have the version, so just `VSCodium.app` (when we make the zip file is when the version gets added to the file name)

Reply:
Let me know if the version tag there should be dropped.